tox.ini

```diff
@@ -72,21 +52,22 @@ commands =
 [testenv:release]
 skip_install = True
 deps =
-	wheel
+	pep517>=0.5
```

Comment:
Today it's already safe to have the python-build CLI instead, especially, given that pep517 drops its own CLI helpers.

Reply:
Yes, probably. And that should be dealt with outside the scope of this PR. jaraco/skeleton#30